client/js/components/assessments/two_button_nav.jsx

```diff
@@ -0,0 +1,95 @@
+"use strict";
+
+import React from "react";
+
+import Button from "../common/button";
+
+export const SECONDARY_ACTION = {
```

Comment:
This would probably be better as PREV_ACTION = {DISABLED, ENABLED} or SECONDARY_ACTION = {PREV, NONE}, or similar.